ports/psoc6/modules/os/moduos.c

```diff
@@ -60,6 +62,22 @@ STATIC mp_obj_t mp_uos_getenv(mp_obj_t var_in) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(mp_uos_getenv_obj, mp_uos_getenv);
 
 
+STATIC mp_obj_t mp_uos_putenv(mp_obj_t var, mp_obj_t value) {
```

Comment:
me not understand? -> This code is to get set environment variables?

Reply:
Yup, can be used for multithreading environments once we have set them up properly.